Scripts/Editor/AssetBundleWindow.cs

```diff
@@ -126,7 +234,57 @@ void OnGUI()
                     CreateAssetBundles.ReadBundleControlFile(Application.dataPath + exportLocation + CreateAssetBundles.bundleControlFileName, bundleVersions);
                     CreateAssetBundles.ReadBundleContentsFile(Application.dataPath + exportLocation + CreateAssetBundles.bundleContentsFileName, bundleContents);
                     ReadBundleFileSizes();
+
+                    //move all .unity3d files into directories named after the title before the .unity3d extension
+                    string[] packsInFolder = Directory.GetFiles(Application.dataPath + exportLocation);
+                    foreach(string file in packsInFolder)
+                    {
+                        if (file.EndsWith(".unity3d"))
+                        {
+                            Regex r = new Regex(@".+[/\\]+");
+                            string name = r.Replace(file.Replace(".unity3d", ""), "");
+
+                            //Create Directory
+                            Directory.CreateDirectory(Application.dataPath + exportLocation + name);
+
+                            //Move unity3d file and rename it to source.unity3d
+                            File.Move(file, Application.dataPath + exportLocation + name + "/source.unity3d");
+
+                            string json = "{\"name\":\"" + name + "\"}";
+                            File.WriteAllText(Application.dataPath + exportLocation + name + "/config.json", json);
+
+                            //Create images
+                            try
+                            {
+                                if (!name.Equals("source")) { 
+                                    File.WriteAllBytes(Application.dataPath + exportLocation + name + "/preview.png", imgPreviews[name]);
+                                }
+                            }
+                            catch(Exception e)
+                            {
+                                Debug.LogWarning(e);
+                            }
+                        }
+                    }
+                    
+                    //create config.json files in all directories containing {"name":"directoryName"}
+                }
+
+
+                //Delete temporary folders
+                string[] needToDeleteDirs = Directory.GetDirectories(Application.dataPath + assetBundleFolderLocation);
+                foreach(string dir in needToDeleteDirs)
+                {
+                    if (!dir.EndsWith("/source"))
+                    {
+                        AssetDatabase.DeleteAsset("Assets" + dir.Replace(Application.dataPath,""));
+                        //Directory.Delete(dir);
+                        //File.Delete(dir + ".meta");
+                    }
```

Comment:
Deletes the folders created earlier in the /AssetBundles Directory as to not crowd up the assets window in unity